docs/content.zh/docs/dev/table/common.md

```diff
@@ -224,7 +224,7 @@ import org.apache.flink.table.api.EnvironmentSettings
 import org.apache.flink.table.api.bridge.scala.StreamTableEnvironment
 
 val bsEnv = StreamExecutionEnvironment.getExecutionEnvironment
```

Comment:
remove `bs` and `b_s_settings` etc. here and below? I think the "b" stood for Blink in the past?